src/index.js

```diff
@@ -20,15 +19,61 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+const operatorFunction = {
+  '+': (x, y) => x + y,
+};
+
+function calculate(fistNumber, secondNumber, operator) {
+  return operatorFunction[operator](fistNumber, secondNumber);
+}
+
+function render({ number, result, operator }) {
+  const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+
+  const operators = ['+', '-', '*', '/'];
+
+  function handleClickNumber(value) {
+    render({
+      number: number * 10 + value,
+      result,
+    });
+  }
+
+  function handleClickOperators(value) {
```

Comment:
```suggestion
  function handleClickOperator(value) {
```

복수개의 `operator` 를 클릭하는게 아니기 때문에 단수가 맞을 것 같아요.